routes/uploads.js

```diff
@@ -24,13 +27,25 @@ function insertImages (db, scene, userID, callback) {
       url: url,
       status: 'initial',
       user_id: userID,
-      messages: []
+      messages: [],
+      metadata: {
+        acquisition_end: scene.acquisition_end,
+        acquisition_start: scene.acquisition_start,
+        contact: `${scene.contact.name},${scene.contact.email}`,
+        platform: scene.platform,
+        provider: scene.provider,
+        properties: {
+          license: scene.license,
+          sensor: scene.sensor
+        },
+        title: scene.title
+      }
     };
   }), callback);
 
   // replace the urls list with a list of _id's
+  // NOTE this causes side-effects
```

Comment:
I don't _think_ these side-effects (which have always been present) are problematic, but they're something to look out for.